src/scripts/ci/publish.ts

```diff
@@ -348,17 +348,20 @@ async function getNewIntegrationVersion(
   console.log(`getNewIntegrationVersion: Next minor stable: ${nextStable}`)
 
   const branchWithoutPrefix = branch.replace(/^integration\//, '')
+  const versionNameSlug = `${nextStable}-integration-${slugify(
```

Comment:
The slugify was missing in one place and it was impossible to deploy a version 2 (could not find the first iteration) and then was failing during npm publish because it was there.

Source of the problem, I had a 'P' uppercase in the name 🔥